### DISCUSSION TITLE
setup-and-deploy-graphql-server

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#3] [Blog Comments]
# setup-and-deploy-graphql-server

Learn how to set up and deploy a production ready Express GraphQL server in TypeScript that can be used by a client to query and persist data from a Postgres database.

https://www.tomray.dev/setup-and-deploy-graphql-server

**Comment by dimasmiftah** [comment]
Thank you so much. It's really helpful! :love:

**Reply by tomwray13** [reply]
You're welcome!

**Comment by asukakuwahara** [comment]
This was exactly what I was looking for and saved so much of my time! Thank you so much for this article!!!!
